### PR TITLE
Feature version 2016 06 22

### PR DESCRIPTION
**Pull request by y2chaits**

### CODE REVIEW

File: src/main/java/com/wepay/model/data/AddressData.java [hunk 1]
```diff
@@ -26,4 +26,15 @@ public static JSONObject buildAddress(AddressData info) throws JSONException {
 		return o;
 	}
 
+	public static JSONObject buildUnifiedAddress(AddressData info) throws JSONException {
```
Comment (JacobKessler) [inline src/main/java/com/wepay/model/data/AddressData.java]: Should this become the default buildAddress and have the old one be renamed to buildLegacyAddress or something like that? Why would someone on a new SDK want an old address?

Comment (y2chaits) [inline src/main/java/com/wepay/model/data/AddressData.java]: Not yet. The unified address format will become the standard in the next version of the SDK. The version of the WePay API we're using here still uses the older address format, except in one case.

**Comment by biplap-sarkar** [comment]
+1

**Comment by skyzyx** [comment]
Reviewed 15 of 15 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/wepay/wepay-java-sdk/42)*

**Comment by skyzyx** [comment]
OK

---

Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/wepay/wepay-java-sdk/42#-:-KPPBCqGiZ1wxcHBFWYy:b32w0x)*

**Comment by skyzyx** [comment]
:lgtm:

---

Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/wepay/wepay-java-sdk/42#-:-KPPBNbZwnKaITY8H0oD:bnfp4nl)*

**Pull request merged**